src/components/Proposals/ProposalCardGroup.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {
+  CardLayout,
+  Tag,
+  GU,
+  textStyle,
+  unselectable,
+  useLayout,
+  useTheme,
+} from '@aragon/ui'
+
+const ProposalCardGroup = ({ title, count, children }) => {
+  const theme = useTheme()
+  const { layoutName } = useLayout()
+  const compactMode = layoutName === 'small'
+  const rowHeight = compactMode ? null : 294
+
+  return (
+    <section>
+      <h2
+        css={`
+          display: flex;
+          align-items: center;
+          margin-bottom: ${3 * GU}px;
+          ${compactMode ? `padding: 0 ${2 * GU}px;` : ''}
```

Comment:
Tiny nitpick but we could the `&&` operator to avoid the empty string.

Reply:
Just a note on this: using `&&` would be interpreted as `"false"` when `compactMode` is `false`, which can create issues with the way the CSS block gets interpreted − especially since the `;` would be missing here. That’s why we always use the ternary operator rather than `&&` (except if the condition itself is a string of course) :smile:

Reply:
Oh that's interesting, I didn't realise that was the case! I assumed styled components was smart enough to strip it out! shall commit that to memory :)

Reply:
Yes I was surprised too the first time. I think they should remove values like `null`, `true`, `false`, similar to how JSX does it. Actually I haven’t checked in a while, so it might be worth checking that it’s still the case!